index.html

```diff
@@ -150,5 +150,43 @@ <h5>Skills</h5>
       </div>
     </div>
   </section>
+  <section class="contact-form">
+    <div id="bg-contact-img1"> <img src="./images/image3.png" alt=""></div>
+  <div id="bg-contact-img"></div>
+  <h2>Get Started</h2>
+  <div><p>Lorem ipsum, dolor sit amet consectetur adipisicing 
+    elit. Sint placeat dolorum perspiciatis rerum 
+    molestiae dicta aperiam libero quod officiis odio.</p></div>
+    <form action="https://formspree.io/f/mpzbbknz" method="POST">
+        <div class="form-dir">
+      <label for="user_name" id="name"> <span>Name</span> 
+        <input class="input-field" type="text" placeholder="Enter your name" id="user_name" required maxlength="30">
+      </label>
+
+     <label for="user_email">E-mail<input type="email" id="user_email" placeholder="Enter your email " required maxlength="30">
+      </label>
+      </div>
+       <textarea name="text" id="user-email" cols="34" rows="10" placeholder="Enter your message here" required maxlength="500"></textarea>
+       <button type="submit">Start Colaboration</button>
+       <img  src="./images/Vector1.png" id="btn-img"  alt="">
+    </form>
+</section>
+<footer>
+  <div id="resume-img"> 
+      <h6>Get my resume </h6>
+    <img  id="cross-img" src="./images/ic_link_teal.png" alt="">
+  </div>
+  <div id="footer-img">
+      <div id="footer-img-left">Get my cv<img src="./images/ic-link_teal.png" alt=""></div>
```

Comment:
- [ ] Finally, add some space on top of the 'get cv' link. Good luck with that 🙌🏼

Reply:
done